payments/schema/types.py

```diff
@@ -0,0 +1,61 @@
+import graphene
+from graphene_django import DjangoConnectionField, DjangoObjectType
+
+from payments.enums import PriceUnits
+from payments.models import BerthPriceGroup, BerthProduct, TAX_PERCENTAGES
+from users.decorators import view_permission_required
+from utils.schema import CountConnection
+
+PriceUnitsEnum = graphene.Enum.from_enum(
+    PriceUnits, description=lambda e: e.label if e else ""
+)
+
+
+class BerthPriceGroupNode(DjangoObjectType):
+    name = graphene.String(required=True)
+    products = DjangoConnectionField("payments.schema.BerthProductNode", required=True)
+    default_product = graphene.Field(
+        "payments.schema.BerthProductNode",
+        description="Returns the first product associated to the price group "
+        "that does not belong to any specific `Harbor`.",
+    )
+
+    class Meta:
+        model = BerthPriceGroup
+        interfaces = (graphene.relay.Node,)
+        connection_class = CountConnection
+
+    @classmethod
+    @view_permission_required(BerthPriceGroup)
+    def get_queryset(cls, queryset, info):
+        return super().get_queryset(queryset, info)
+
+    def resolve_products(self, info):
+        return self.products.all()
```

Comment:
if this is all this resolver does, it is kinda redundant, as this is the default logic anyway

Reply:
Seems to work 👍 it was probably a (functioning) leftover from some previous tests. Gone now.